ui/src/funding/contract.ts

```diff
@@ -1,25 +1,61 @@
-import type { BigNumber, ContractTransaction, Signer } from "ethers";
+import type { ethers, BigNumber, ContractTransaction, Signer } from "ethers";
+
+import Big from "big.js";
+
+import * as svelteStore from "svelte/store";
+
 import {
   Erc20Pool,
   Erc20Pool__factory as PoolFactory,
   ERC20,
   ERC20__factory as Erc20Factory,
 } from "radicle-contracts/build/contract-bindings/ethers";
 
-// TODO(nuno): make the contract addresses configurable/env-dependant
-
-export const POOL_ADDRESS: string =
-  "0x37c10d847bf9e708add451bf2f80c1297d7aa691";
+import * as ethereum from "../ethereum";
+
+const addresses = {
+  pool: {
+    local: "0x37c10d847bf9e708add451bf2f80c1297d7aa691",
+    ropsten: "0xEc5bfca987C5FAA6d394044793f0aD6C9A85Da76",
+  },
+  dai: {
+    local: "0xf34a89802590f944e3de71b1f74d66ed1bafc9cd",
+    ropsten: "0xD069f9Cbe64979953357bCa3f21d902e775f1F48",
+  },
+};
```

Comment:
Since the contracts are immutable, and since pointing to a proxy address that resolves to the latest one might cause API breaks at this stage of the game, I'd figure we could hardcode them for now. We can discuss introducing proxy contracts as a follow-up issue.